functions/go/set-annotations/annotations_transformer_test.go

```diff
@@ -191,3 +193,62 @@ spec:
 		t.Fatalf("Actual doesn't equal to expected")
 	}
 }
+
+func TestAnnotationsTransformerResults(t *testing.T) {
+	config := `
+annotations:
+  app: myApp
+`
+	input := `
+apiVersion: v1
+kind: Service
+metadata:
+  name: myService
+spec:
+  ports:
+  - port: 7002
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mungebot
+  labels:
+    app: mungebot
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: mungebot
+    spec:
+      containers:
+      - name: nginx
+        image: nginx
+`
+	expectedResults := []*Result{
+		{
+			FilePath:  "",
```

Comment:
You can set path annotations on resources to test this part.

Reply:
I didn't know about that, thanks for pointing this out. Done